### ISSUE TITLE
Wrong detection of text wrapping in a cell

### ISSUE THREAD

**Issue by Akrobs** [ONLYOFFICE/DocumentServer#2137]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DocuemntServer/issues?q=) and did not find an issue describing my bug.

### Operating System of DocumentServer

Linux (RPM package)

### Version information

7.3.2

### Expected Behavior

Not sure.

### Actual Behavior

If you open a spreadsheet document, the text wraps in the first cell, but it shouldn't wrap. Compare screenshots from Excel and OO.

### Reproduction Steps

1) Open attach doc in OO
2) Open attach doc in MS
3) Compare first cell
[TF02802352.xlsx](https://github.com/ONLYOFFICE/DocumentServer/files/10853033/TF02802352.xlsx)

### Additional information

<img width="960" alt="MS Excel" src="https://user-images.githubusercontent.com/57731196/221928155-f03027df-bc35-49bc-b4af-7cfbb4aad2e7.png">
<img width="960" alt="OO-Default" src="https://user-images.githubusercontent.com/57731196/221928165-03e77562-c73a-479e-a639-a6e85a4a4c4c.png">
<img width="959" alt="OO-off wrap" src="https://user-images.githubusercontent.com/57731196/221928175-5af73cb7-b34a-441c-9c50-8a1de99a4762.png">

**Comment by rsnegirev** [comment]
Hello, @Akrobs 

I can confirm - this is the bug, issue 61480 in our internal issue tracker.
Thank you for the report.

**Comment by Akrobs** [comment]
7.5.1 bug is still present (:

**Comment by rsnegirev** [comment]
@Akrobs yes, unfortunately that true. I'll raise the bug version and remind the developers.

**Comment by Akrobs** [comment]
Thanks!

**Comment by Gingetsu** [comment]
Hello @Akrobs! I have a good news! This issue will be fixed in the next release (https://github.com/ONLYOFFICE/sdkjs/commit/d28f3c5c29035dc069a90dfcd5fe4fb5a3ac58db). Please keep checking for updates.

**Comment by Rita-Bubnova** [comment]
DocumentServer v8.3.0 is released so I close this issue. Feel free to comment or reopen it if you got further questions.
